### PR TITLE
#424 Integrate new Option model to TableEditor

### LINKED ISSUE #424: admin.py:178: Adapt admin panel to Options
The puzzle `396-6d2bc254` from #396 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/196e6cb4f653274e81ff1cdef748345a721846a2/stroyprombeton/admin.py#L178-L178

The puzzle was created by duker33 on 30-Jan-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by artemiy312**
Closes #424

### CODE REVIEW

**Comment by artemiy312** [comment]
This seems to doesn't fit in Table editor arch, because Product has the 1:n relation with Option, so we have to either leave only one Option's field to set relations with a product or separate table editor to Option table editor and Product table editor to keep editing feature of Option's fields.

**Comment by artemiy312** [comment]
@duker33 What do you think at this point?

**Comment by duker33** [comment]
@artemiy312 , we'll integrate Tags after Options and we'll resurrect Table Editor there: https://github.com/fidals/stroyprombeton/issues/422

If we'll implement Options now, we'll throw them away after tags integration

File: front/js/admin.es6 [hunk 1]
```diff
@@ -27,7 +71,7 @@ const customColModels = [
     width: 50,
   },
   {
-    name: 'length',
+    name: 'options_length',
```
[the remark this answers is not in the document]
Comment (artemiy312) [reply front/js/admin.es6]: It was supposed to show `Option.length` field, but it was a mistake, so I partially reversed these changes for #422

**Pull request merged**